config.go

```diff
@@ -114,6 +114,7 @@ type config struct {
 	ConnectPeers         []string      `long:"connect" description:"Connect only to the specified peers at startup"`
 	CPUProfile           string        `long:"cpuprofile" description:"Write CPU profile to the specified file"`
 	MemoryProfile        string        `long:"memprofile" description:"Write memory profile to the specified file"`
+	TraceProfile         string        `long:"trace" description:"Write execution trace to the specified file"`
```

Comment:
nit: call the flag `traceprofile` to not confuse it with a log level and be more in line with the other profile flags?

Reply:
Makes sense ^

Reply:
Bump on this, can get merged once addressed.

Reply:
Addressed in the latest push